### ISSUE TITLE
Unable to open projects saved with VST3 on Windows with Mac Os

### ISSUE THREAD

**Issue by axtTOKQf** [DISTRHO/Cardinal#612]
I'm using Ableton Live 11.3.21 and Cardinal 23.10 VST3 Synth version and when I try to open the same project on Windows 10 with the same version numbers, it says "couldn't find the vst" even though there is a Cardinal Synth in the plugin list.

**Comment by dromer** [comment]
Can you load that plugin into a new session?

Sounds to me like it could be an Ableton bug though.

**Comment by axtTOKQf** [comment]
Yes, I can load the plugin. It seems like they have different "ID's" somehow.

**Comment by minim-sheba** [comment]
I am finding the same problem with moving an Ableton set that uses Cardinal FX between Windows 11 and Mac OS Sequoia. 

The file I first noticed this in was created in Ableton Live 11 on Windows 11, but I upgraded to Ableton Live 12 on both computers when I got the Mac in March. When I opened the file on the Mac, it seemed to convert to 12 OK, but I received an error "The VST3 plug-in could not be found" on every instance of Cardinal FX in the set. However, it's clearly finding the plugin - I can see it in the list and even drag a new instance into a track and start working with that, it's just apparently not able to associate the plugin it can see with the information about the plugin it finds in my file from Windows. I was able to export the settings for the plugin on Windows and open them in the newly inserted Cardinal FX instance on Mac and then everything seems to work fine - except when I saved the file and took it back to Windows, then Windows shows the "VST3 plug-in could not be found error"- but only on the instance of Cardinal FX added on the Mac - it has no problem with the original plug-in instances.

The same behaviour is exhibited when adding the plugin in a brand new file - the VST3 added on Mac can't be seen by Windows and vice versa.

I've tried deleting and then reinstalling Cardinal, reinstalling Live, and also tried downgrading to 24.09 which is the version the set was created with, but none of these fix the problem.

Cardinal version: 24.12 (64-bit Windows and Mac Universal)
Ableton Live Standard 12.1.11
Windows 11 Home, Version 10.0.26100 Build 26100, AMD Ryzen 9 - reinstalled from scratch about a month ago
Mac OS Sequoia 15.3.1, MacBook Air M1 (2020) - reinstalled from scratch about 6 weeks ago

**Comment by kawa-sanmyaku** [comment]
Good evening, I was just about to open an issue but found this one; it does seem as though the VST3 variants of Cardinal have different IDs between the Linux and Windows versions as well

To test this, I have a Linux host and a Windows virtual machine, both with REAPER 7.47; both are of course loading their respective native binaries

I created a project with four tracks - one with CardinalSynth (VST2), one with the main VST3 variant, one with the main LV2 variant, and the last one with the main CLAP variant; I saved the project on REAPER in the Linux host, and opened the project on REAPER in the Windows virtual machine

Only the VST3 variant failed to load :( the project doesn't care about whether the plugins that were loaded were the Linux or Windows variants, it only looks for matching IDs

<img width="1920" height="933" alt="Image" src="https://github.com/user-attachments/assets/f5822805-0d81-4fba-89c1-ee6b6b561f51" />

**Comment by dromer** [comment]
@kawa-sanmyaku This macro should actually be blocked on VST3 using `DPF_VST3_DONT_USE_BRAND_ID` <https://github.com/DISTRHO/Cardinal/blob/main/src/Cardinal/DistrhoPluginInfo.h#L65>
